_data/chains/eip155-28.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "OMG Network Testnet Rinkeby",
+  "chain": "ETH",
+  "network": "rinkeby",
+  "rpc": ["https://rinkeby.omgx.network/"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Rinkeby Ether",
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "icon": "omgx",
+  "infoURL": "https://www.omgx.network/",
+  "shortName": "omgx",
+  "chainId": 28,
+  "networkId": 28,
+  "explorers": [{
+    "name": "etherscan",
```

Comment:
pretty sure this is not etherscan ;-)
also the block explorer did not load for me - so I could not check if it is really EIP3091 compatible

Reply:
whooops, indeed!
@ligi block explorer should be up now, thank you for checking!